lib/handlers/compile.ts

```diff
@@ -206,7 +206,8 @@ export class CompileHandler {
                 compiler.exe = exe;
             } else {
                 // errors resolving to absolute path are not fatal for backwards compatibility sake
```

Comment:
I wasn't sure whether this comment is referring to "exit CE" fatal or "don't create a compiler" fatal. The former is still the case (though isn't really for backwards compatibility), while the latter isn't any more.

Reply:
I was also confused by this comment so do not worry

Reply:
It refers to still creating the compiler if not found, this is mostly because the `which()` call was added later on and/or it was not completely understood yet if `which()` covered all the use-cases.

I think the only thing that `which` cannot detect is builtin commands. Say you would want to use `echo` as a compiler, that would have worked before, and will not anymore now with the `return null`. I think that situation is fine.

Things like `.bat` and `.cmd` files are correctly detected by `which`, so in those cases it can detect the absolute path correctly.

So unless I'm missing a use-case, I think this change is Ok.

Reply:
(feel free to remove the comment)